runtime/renderer/injected_bundle.cc

```diff
@@ -55,7 +55,7 @@ class BundleGlobalData {
     common::ApplicationData* app_data =
         appdata_manager->GetApplicationData(app_id);
 
-    app_data->LoadManifestData();
```

Comment:
How about check about already loaded in LoadManifestData

example :
bool ApplicationData::LoadManifestData() {
   if (loaded_) {
      return true;
   }
   loaded = true;
  ...
}